src/modules/phone/index.ts

```diff
@@ -51,11 +55,21 @@ export class Phone {
   /**
    * Returns a random phone number format.
    *
+   * @see faker.phone.phoneNumber
+   * @see faker.definitions.phone_number.formats
+   *
    * @example
    * faker.phone.phoneFormats() // '!##.!##.####'
+   *
+   * @deprecated
    */
-  // TODO @pkuczynski 2022-02-01: simplify name to `format()`
   phoneFormats(): string {
+    deprecated({
+      deprecated: 'faker.phone.phoneFormats',
+      proposed: 'faker.phone.phoneNumber',
```

Comment:
Previously we used `()` like:
```suggestion
      deprecated: 'faker.phone.phoneFormats()',
      proposed: 'faker.phone.phoneNumber()',
```
I'm fine with both :shrug: just pops up in my mind